desktop/settings/mac.md

```diff
@@ -176,7 +176,7 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
 
 ### Network
 
-You can configure Docker Desktop networking to work on a virtual private network (VPN). Specify a network address translation (NAT) prefix and subnet mask to enable Internet connectivity.
+Docker Desktop uses a private IPv4 network for internal services such as a DNS server and an HTTP proxy. In case the choice of subnet clashes with something in your environment, specify a custom subnet using the **Network** setting.
```

Comment:
```suggestion
Docker Desktop uses a private IPv4 network for internal services such as a DNS server and an HTTP proxy. In case the choice of subnet clashes with your environment, you can specify a custom subnet using the **Network** setting.
```

The description was perhaps slightly vague unnecessarily?